tests/other_errors/funcptr_method.jou

```diff
@@ -0,0 +1,8 @@
+class Foo:
+    def bar(self) -> None:
+        pass
+
+
+def foo() -> None:
+    f = Foo{}
+    thing = f.bar  # Error: Foo.bar() is a method, it can be called but it cannot be used in any other way
```

Comment:
I wrote another review comment about adding a test like this. It already exists and both of these two review comments can be ignored.